arangod/Aql/RemoteExecutor.cpp

```diff
@@ -503,8 +503,8 @@ Result ExecutionBlockImpl<RemoteExecutor>::sendAsyncRequest(fuerte::RestVerb typ
   req->timeout(kDefaultTimeOutSecs);
   if (!_distributeId.empty()) {
     req->header.addMeta("x-shard-id", _distributeId);
```

Comment:
Maybe out of scope here but we should stick to static strings everywhere. As it is newly implemented in L507